<launch>
  <arg name="LAUNCH_DYNAMIC_TF_PUBLISHER" default="true" />
  <arg name="models" default="$(find jsk_interactive_marker)/launch/models/urdf_model_markers.yaml" />
  <node pkg="jsk_interactive_marker" type="move_base_marker" name="jsk_move_base_marker" output="screen" >
    <rosparam command="load" param="model_config" file="$(arg models)"/>
    <param name="use_dynamic_tf" value="$(arg LAUNCH_DYNAMIC_TF_PUBLISHER)"/>
  </node>

  <group if="$(arg LAUNCH_DYNAMIC_TF_PUBLISHER)">
    <node pkg="dynamic_tf_publisher" type="tf_publish.py" name="dynamic_tf_publisher_for_move_base_marker" output="screen">
      <param name="use_cache" type="bool" value="false"/>
    </node>
  </group>
</launch>